### PR TITLE
Feature/ipset

### PR DESCRIPTION
**Pull request by fdurand**
for review

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -13,6 +13,13 @@ For more details and developer visible changes see the ChangeLog file.
 For a list of compatibility related changes see the UPGRADE file.
 
 --------------------------------------------------------------------------------
+Version <devel> released on <releaseDate>
+
+New Features
+ * Adding ipset support for inline mode
+ * Adding support for multivlan in inline mode
+ * Adding support of portal profil by vlan
```
Comment (dwlfrth) [inline NEWS]: s/profil/profiles

File: docs/PacketFence_Administration_Guide.asciidoc [hunk 2]
```diff
@@ -304,6 +304,11 @@ server, DHCP server, RADIUS server) using:
   sudo apt-get update
   sudo apt-get install packetfence
 
+In order to use ipset in inline mode, you must install two news dependencies and compile kernel modules:
```
Comment (dwlfrth) [inline docs/PacketFence_Administration_Guide.asciidoc:310]: That is the Debian based doc... is there anything for RHEL?

Comment (fdurand) [inline docs/PacketFence_Administration_Guide.asciidoc:310]: For RHEL5 no way to use ipset , for RHEL6 , ipset is in the spec

Comment (obilodeau) [inline docs/PacketFence_Administration_Guide.asciidoc:310]: It should be documented as such and the NEWS entry should mention it too. I would probably do it in a ipset dedicated section. Not really sure where to put it though..

**Comment by obilodeau** [comment]
Does this fixes #1424 for all supported platforms? http://packetfence.org/bugs/view.php?id=1424

File: debian/control [hunk 3]
```diff
@@ -16,6 +16,8 @@ Depends: ${misc:Depends}, apache2, apache2.2-common, apache2-utils,
  openssl, openssl-blacklist, openssl-blacklist-extra, php-log, snort, 
  mysql-server, libapache2-mod-proxy-html, libapache2-mod-php5, php-pear,
  php5-mysql, php5-gd, perl-suid, make,
+# ipset (ipset must be installed by hand  apt-get install xtables-addons-common  xtables-addons-source)
```
Comment (obilodeau) [inline debian/control:19]: installed by hand? reason?

**Comment by fdurand** [comment]
Debian , centos 6 and RHEL6, not centos 5

File: debian/control (hunk 3 above)
Comment (fdurand) [inline debian/control:19]: You must have to compil the kernel module by module-assistant auto-install xtables-addons.
So when you install packetfence with apt-get you can´t build the module in the init script (it use dpkg that is already locked).
So i prefer to document that and install by hand.

Comment (obilodeau) [inline debian/control:19]: and how will that be handled in the liveusb creation process?

**Comment by obilodeau** [comment]
re: not on centos 5

Then one of the alternatives we discussed back in june will have to be considered. Unless we are willing to drop support for it in our next feature release. Also, I'm not sure if I buy the sudoers idea. I opposed to it earlier but you still took that direction...

File: debian/control (hunk 3 above)
Comment (fdurand) [inline debian/control:19]: The kernel module must be build the first time the live usb is running.
And i have a problem with installing xtables-addons-common  xtables-addons-source during the build, it remove packetfence.
So by hand.

**Comment by fdurand** [comment]
It´s also be possible to implement ipset on centos 5, but we have to make kernel module for at least some of the last kernel version.
For sudoers ,i was inspired by perl Captive::Portal that i think is the better solution

**Comment by fdurand** [comment]
Warning on restart:
http://www.packetfence.org/bugs/view.php?id=1539

**Comment by dwlfrth** [comment]
Do we need to keep this one opened since the whole thing have been rewritten object style?

**Comment by fdurand** [comment]
no, but keep this branch on github yes, it has been deployed in some clients network

**Comment by dwlfrth** [comment]
Closed since this has been reworked on an object oriented base
#59

**Pull request closed without merging**